docs/installation/distribution.rst

```diff
@@ -57,3 +57,9 @@ And bdgenomics.mango:
 .. code:: bash
 
     pip install bdgenomics.mango
+
+And bdgenomics.adam:
```

Comment:
This shouldn't be necessary, bdgenomics.adam is a dependency of bdgenomics.mango

Reply:
Sounds good, double checked the pip install and removed these lines.